experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/AndroidImageStorage.kt

```diff
@@ -0,0 +1,63 @@
+package example.imageviewer
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.core.graphics.scale
+import example.imageviewer.model.PictureData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+private const val maxStorableImageSize = 2000
+private const val storableThumbnailSize = 200
+
+class AndroidImageStorage(
+    private val pictures: SnapshotStateList<PictureData>,
+    private val ioScope: CoroutineScope
+) : ImageStorage {
+    val mapWithBigImages = mutableMapOf<PictureData.Camera, ImageBitmap>()
+    val mapWithThumbnails = mutableMapOf<PictureData.Camera, ImageBitmap>()
```

Comment:
Another naming comment) Usually it's a bad practice to mention data type in name
```suggestion
    val largeImages = mutableMapOf<PictureData.Camera, ImageBitmap>()
    val thumbnails = mutableMapOf<PictureData.Camera, ImageBitmap>()
```

Reply:
Done